internal/ingress/store/store.go

```diff
@@ -160,11 +164,30 @@ func (s Store) GetService(namespace, name string) (*apiv1.Service, error) {
 	return service.(*apiv1.Service), nil
 }
 
-// ListIngresses returns the list of Ingresses
-func (s Store) ListIngresses() []*networkingv1beta1.Ingress {
+// ListIngressesV1 returns the list of Ingresses in the Ingress v1 store.
+func (s Store) ListIngressesV1() []*networkingv1.Ingress {
+	// filter ingress rules
+	var ingresses []*networkingv1.Ingress
+	for _, item := range s.stores.IngressV1beta1.List() {
```

Comment:
Is the store being used correct? Should it be `s.stores.IngressV1.List()` instead?

Reply:
🤦. Thanks for catching this! This is a bug which I made both in the code (`Store`) and, symmetrically, in the test (`FakeStore`).

- Added an extra test (asserting that v1 ingress store is empty if we added v1beta1, and vice versa) that would've caught this.
- Also, this is a good example why tests should have minimal logic.